virocon/plotting.py

```diff
@@ -60,6 +125,23 @@ def ppf(self, q):
     return axes
 
 def plot_dependence_functions(model, model_desc=None, par_rename=None, axes=None):
+    """
+    Function to plot the dependence functions of the indicated model.
+       
+    Parameters
+    ----------
+    model :  MultivariateModel
+        The model to be used to plot the dependence functions.      
+    model_desc: dict, optional
+        Generated model description. Defaults to None.
+    par_rename : optional
+        ?
```

Comment:
dict, optional
A dictionary that maps from names of conditional parameters to a string. If e.g. the model has a distribution with a conditional parameter named 'mu' one could change that in the plot to '$mu$' with {'mu': '$mu$'}.